SensorIPAddress_0: “10.13.1.113” #在出厂时，雷达传感器将配置一个默认 IP地址 10.13.1.113。注意更改 IP 地址后需要重启雷达传感器。
SensorIPAddress_1: “169.254.116.113” #注意第二个 IP 地址（SensorIPAddress_1）目前没有使用，应该保持其默认值(169.254.116.113)。非常重要的是：第二个 IP 地址(169.254.116.113)也必须在相同的周期内发送配置。否则，真实 IP 10.13.1.113 不会发生变化。
